README.md

```
@@ -116,10 +116,10 @@ For detailed guides, command options, and advanced usage examples, please check
 ## Labs
 
 <p align="center">
-  <img src="https://github.com/shriyanss/js-recon-labs/blob/main/static/labs-banner.png?raw=true" alt="JS Recon Labs" width="300"/>
+  <img src="https://github.com/js-recon/js-recon-labs/blob/main/static/labs-banner.png?raw=true" alt="JS Recon Labs" width="300"/>
 </p>
 
-Labs to test JS Recon tool are available [here](https://github.com/shriyanss/js-recon-labs). Walkthroughs are available [here](https://js-recon.io/labs).
+Labs to test JS Recon tool are available [here](https://github.com/js-recon/js-recon-labs). Walkthroughs are available [here](https://js-recon.io/labs).
```

markdownlint-cli2 (0.23.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)